Sources/KituraNet/HTTP/HTTPServerResponse.swift

```diff
@@ -180,11 +156,26 @@ public class HTTPServerResponse: ServerResponse {
 
         //We don't keep the connection alive on an HTTP error
         headers["Connection"] = ["Close"]
-        let response = HTTPResponseHead(version: HTTPVersion(major: 1, minor: 1), status: status, headers: headers.httpHeaders())
+
+        execute(on: channel.eventLoop) {
+            do {
+                try self.sendResponse(channel: channel, handler: handler, status: status, withBody: withBody)
+            } catch let error {
+                fatalError("Error: \(error)")
+            }
+        }
+    }
+
+    /// Send response to the client
+    private func sendResponse(channel: Channel, handler: HTTPRequestHandler, status: HTTPResponseStatus, withBody: Bool = false) throws {
```

Comment:
`sendResponse` is going to be used for both normal and error responses. So, it makes sense to have `withBody` defaulting to `true`. We can assume that most of the responses do have a body.

Reply:
@pushkarnk Thanks! I have addressed the changes